pkg/app/piped/cmd/piped/piped.go

```diff
@@ -616,8 +616,14 @@ func (p *piped) insertLoginUserToPasswd(ctx context.Context) error {
 	return nil
 }
 
-func registerMetrics() {
+func registerMetrics(pipedID string) {
 	r := prometheus.DefaultRegisterer
+	// TODO: Add piped version as label.
+	prometheus.WrapRegistererWith(
+		prometheus.Labels{"piped": pipedID},
+		r,
```

Comment:
Nice. One thing we have to update is fixing `cli` to use this wrapped register instead of the default one.
https://github.com/pipe-cd/pipe/blob/master/pkg/cli/cmd.go#L121

But that can be done in another PR.

Reply:
Nice catch, lets me address it in this PR 🙏

Reply:
addressed by https://github.com/pipe-cd/pipe/pull/2181/commits/0eb829f433fe0e6335dcf6dac5aab1670e31d178 🙏